cpp/include/cudf/strings/convert/convert_fixed_point.hpp

```diff
@@ -0,0 +1,124 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace cudf {
+namespace strings {
+/**
+ * @addtogroup strings_convert
+ * @{
+ * @file
+ */
+
+/**
+ * @brief Returns a new fixed-point column parsing decimal values from the
+ * provided strings column.
+ *
+ * Any null entries will result in corresponding null entries in the output column.
+ *
+ * Only characters [0-9] plus a prefix '-' and '+' and a single decimal point
+ * are recognized. When any other character is encountered, the parsing ends
+ * for that string and the current digits are converted into a fixed-point value.
+ *
+ * @code{.pseudo}
+ * Example:
+ * s = ['123', '-876', '543.2', '-0.12']
+ * dt = {DECIMAL32, scale=-2}
+ * fp = to_fixed_point(s, dt)
+ * fp is [123400, -87600, 54320, -12]
+ * @endcode
+ *
+ * Overflow of the resulting value type is not checked.
+ * The scale in the `output_type` is used for setting the integer component.
+ *
+ * @throw cudf::logic_error if `output_type` is not a fixed-point decimal type.
+ *
+ * @param strings Strings instance for this operation.
+ * @param output_type Type of fixed-point column to return including the scale value.
+ * @param mr Device memory resource used to allocate the returned column's device memory.
+ * @return New column of `output_type`.
+ */
+std::unique_ptr<column> to_fixed_point(
+  strings_column_view const& input,
+  data_type output_type,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
+/**
+ * @brief Returns a new strings column converting the fixed-point values
+ * into a strings column.
+ *
+ * Any null entries will result in corresponding null entries in the output column.
+ *
+ * For each value, a string is created in base-10 decimal.
+ * Negative numbers will include a '-' prefix.
+ * The column's scale value is used to place the decimal point.
+ * A negative scale value may add padded zeros after the decimal point.
+ *
+ * @code{.pseudo}
+ * Example:
+ * fp is [110, 222, 3330, -440, -1] with scale = -2
+ * s = from_fixed_point(fp)
+ * s is now ['1.10', '2.22', '33.30', '-4.40', '-0.01']
+ * @endcode
+ *
+ * @throw cudf::logic_error if the `input` column is not a fixed-point decimal type.
+ *
+ * @param input Fixed-point column to convert.
+ * @param mr Device memory resource used to allocate the returned column's device memory.
+ * @return New strings column.
+ */
+std::unique_ptr<column> from_fixed_point(
```

Comment:
More generic `to_strings` function needed for other column types?

Reply:
You are welcome to open a feature request to add this new API. Each one usually requires some unique formating data like `true/false` strings for booleans, datetime format specifiers for timestamps. But the others may just take an output data_type.
https://docs.rapids.ai/api/libcudf/nightly/group__strings__convert.html